appcoins-ads/src/main/java/com/asf/appcoins/sdk/ads/poa/manager/WalletUtils.java

```diff
@@ -16,18 +16,22 @@
 import android.net.Uri;
 import android.os.AsyncTask;
 import android.os.Build;
+import android.telephony.TelephonyManager;
 import android.util.Log;
 import com.asf.appcoins.sdk.ads.BuildConfig;
 import com.asf.appcoins.sdk.ads.listeners.CafeBazaarResponseAsync;
 import com.asf.appcoins.sdk.ads.listeners.ResponseListener;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Locale;
 
 public class WalletUtils {
 
   private static final String URL_BROWSER = "https://play.google.com/store/apps/details?id="
       + com.appcoins.billing.sdk.BuildConfig.BDS_WALLET_PACKAGE_NAME;
-  private static final String CAFE_BAZAAR_URL = "bazaar://details?id=com.hezardastan.wallet";
+  private static final String CAFE_BAZAAR_APP_URL = "bazaar://details?id=com.hezardastan.wallet";
+  private static final String CAFE_BAZAAR_WEB_URL =
+      "https://cafebazaar.ir/app/com.hezardastaan.wallet";
```

Comment:
these string maybe should be in build.gradle?

Reply:
The aptoide and google play weren't so I think these shouldn't as well.